SU-HSDO/suhumsci
# Hsd8 396 setup subtheme support

# READY FOR REVIEW

# Summary
- Allow for custom subthemes
- dont export config that is ignored

# Needed By (Date)
- ASAP

# Urgency
- High

# Steps to Test
1. Checkout this branch
1. `composer update --prefer-source`
1. Sync archaeology `blt drupal:sync --site=archaeology`
1. change the theme to the "Stanford HumSci SubTheme"
1. run config export `drush @archaeology.local csex -y`
1. ensure no new configs were exported
1. run config import `drush @archaeology.local cim -y`
1. ensure the new theme is still enabled.


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
